flexmeasures/api/v3_0/sensors.py

```diff
@@ -1575,13 +1622,170 @@ def trigger_forecast(self, id: int, sensor: Sensor, **params):
+              example: b3d26a8a-7a43-4a9f-93e1-fc2a869ea97b
+              schema:
+                type: string
+          responses:
+            200:
+              description: Forecast job status or results
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: ["PENDING", "RUNNING", "FAILED", "FINISHED"]
+                      start:
+                        type: string
+                        format: date-time
+                        description: Start time of the forecast values (ISO 8601).
+                      end:
+                        type: string
+                        format: date-time
+                        description: End time of the forecast values (ISO 8601).
+                      resolution:
+                        type: string
+                        description: Resolution of the forecast values as an ISO-8601 duration.
+                      unit:
+                        type: string
+                        description: Unit of the forecast values.
+                      values:
+                        type: array
+                        items:
+                          type: number
+                        description: Forecast values.
+                  examples:
+                    finished:
+                      summary: Finished forecasting job
+                      value:
+                        start: "2025-10-15T00:00:00+01:00"
+                        end: "2025-10-15T04:00:00+01:00"
+                        resolution: "PT1H"
+                        unit: "kW"
+                        values: [1.2, 1.5, 1.4, 0.8]
+            401:
+              description: UNAUTHORIZED
+            403:
+              description: INVALID_SENDER
+            422:
+              description: UNPROCESSABLE_ENTITY
+          tags:
+            - Sensors
+        """
+
+        try:
+            # Fetch job from RQ
+            queue = current_app.queues["forecasting"]
+            job = Job.fetch(job_id, connection=queue.connection)
+
+            # Job does not exist
+            if job is None:
+                return unprocessable_entity(f"Job {job_id} not found.")
+
+            # Ensure the sensor in the URL matches the sensor associated with the forecast job
+            job_sensor_id = job.meta.get("sensor_id")
+            if job_sensor_id != sensor.id:
+                return unprocessable_entity(
+                    f"Sensor ID mismatch: job {job_id} is for sensor {job_sensor_id}, not sensor {sensor.id}."
+                )
+            if job.dependency_ids:
+                return unprocessable_entity(
+                    f"Job {job_id} is the wrap-up job; please query one of those cycle jobs {job.dependency_ids} instead."
```

Comment:
```suggestion
                    f"Job {job_id} is the wrap-up job; please query one of the cycle jobs {job.dependency_ids} instead."
```

But actually, although we expect most users won't encounter this error, because they'd use single-cycle triggers, instead of returning a 422, I'd prefer a 200 with a `"forecasts": [<uuid>, <uuid>]` key and we could still keep a `"message"` key, too, but then not add a potentially long list inside that string:

```suggestion
                    f"The forecasting job led to forecasts over different periods. Please query an individual period by calling this endpoint with one of the listed job IDs."
```

Reply:
Good suggestion, applied. now we return the jobs_ids and that message with 200 code when wrap-up job is passed